README.md

```diff
@@ -128,6 +129,18 @@ vagrant up sensu-backend-peer1 sensu-backend-peer2
 vagrant provision sensu-backend-peer1 sensu-backend-peer2
 ```
 
+#### Beginning with a Sensu federated cluster
+
+Multiple Vagrant boxes are available for testing a Sensu Go federated cluster.
+First build and provision both then provision the first a second time to view that the custom role was replicated.
+
+```base
+vagrant up sensu-backend-federated1 sensu-backend-federated2
+vagrant provision sensu-backend-federated1
+```
+
+The `provision` command should output from `sensuctl` the `test` Sensu Go Role that was created on the other backend.
```

Comment:
Please provide sample output so users can confirm they are getting the desired effect.

Reply:
Added output that should be printed.